wallet/src/lib.rs

```diff
@@ -218,7 +324,7 @@ mod wallet {
             args.canister.clone(),
             "wallet_receive",
             (),
-            args.amount as i64,
+            args.amount as u64,
```

Comment:
```suggestion
            args.amount,
```
they're already u64 we can just drop the as i64 here

Reply:
done